assets/activation-scripts/etc/profile.d/0800_cuda.sh

```diff
@@ -35,8 +35,10 @@ activate_cuda() {
   fi
 
   # Use system library cache.
-  SYSTEM_LIBS=$("$ldconfig_bin" --print-cache -C /etc/ld.so.cache 2> /dev/null \
-    | "$_nawk/bin/nawk" "\$1 ~ /${lib_pattern}/ { print \$4 }")
+  SYSTEM_LIBS=$(
+    { "$ldconfig_bin" --print-cache -C /etc/ld.so.cache 2> /dev/null || echo; } \
+      | "$_nawk/bin/nawk" "\$1 ~ /${lib_pattern}/ { print \$4 }"
+  )
```

Comment:
Glad you added the test for this below! Did you double check it doesn't fail on actual NixOS? If not maybe Zach should?

Reply:
I don't have access to a NixOS machine with Nvidia hardware. I'm pretty confident in the failure mode, but @zmitchell if you're able to double check that'd be great.